2023/10/20/main.go

```diff
@@ -0,0 +1,16 @@
+// Daily Joel 2023-10-20
+//
+// Arrays and basic for loops.
+package main
+
+import (
+	"fmt"
+)
+
+func main() {
+	letters := [4]string{"J", "o", "e", "l"}
+	for i := 0; i < len(letters); i++ {
+		fmt.Print(letters[i])
```

Comment:
Declaring a for loop with initialisation defined with a variable assigned 0(int).

The condition is a boolean statement that in this case is defined as variable i is smaller than the size of the array. 
As long as the boolean statement is true the for loop will continue. In this case it therefore will stop when the variable i is same size as the array. 

The body of the for loop is a print function where we print the variable letters and its index (variable i) we will therefore print the strings in the array one by one incrementing the index by 1